version/version.sh

```diff
@@ -17,7 +17,7 @@ var Package = "$(go list)"
 // Version indicates which version of the binary is running. This is set to
 // the latest release tag by hand, always suffixed by "+unknown". During
 // build, it will be replaced by the actual version. The value here will be
-// used if the registry is run after a go get based install.
+// used if the registry is run after a go install based install.
```

Comment:
Looks like we didn't update this in the `main` branch; we should do it there as well;
https://github.com/distribution/distribution/blob/92d136e113cf34fcfd5f0c21fadf6eac2f39e803/version/version.sh#LL20

Reply:
will raise pr on main later.